packages/headless/src/features/commerce/search-parameters/search-parameter-serializer.ts

```diff
@@ -0,0 +1,69 @@
+import {CommerceSearchParameters, Parameters} from './search-parameter-actions';
```

Comment:
Mmm... in the end when we support all query params, that'll be a lot of duplication from the original search-parameter-serializer. Why do we need to duplicate this? Could we instead make the current search-parameter-serializer a tad more generic?